client/components/shipping/packages/add-package.js

```diff
@@ -49,10 +49,10 @@ const updateFormTextField = ( event, updatePackagesField ) => {
 	updatePackagesField( { [name]: value } );
 };
 
-const renderOuterDimensions = ( showOuterDimensions, packageData, value, updatePackagesField, is_user_defined ) => {
+const renderOuterDimensions = ( showOuterDimensions, dimensionUnit, packageData, value, updatePackagesField, is_user_defined ) => {
 	return ( showOuterDimensions || packageData.outer_dimensions ) ? (
 		<FormFieldset>
-			<FormLabel>{ __( 'Outer Dimensions (L x W x H)' ) }</FormLabel>
+			<FormLabel>{ __( 'Outer Dimensions (L x W x H)' + ' ' + dimensionUnit ) }</FormLabel>
```

Comment:
This is bad translation practice - it isn't compatible with RTL languages, plus the string would need to be translated for all possible `dimensionUnit` values.

See: https://github.com/Automattic/woocommerce-connect-client/blob/master/client/components/shipping/services/group.js#L8-L17